src/block/block.js

```diff
@@ -59,8 +59,8 @@ registerBlockType( 'cgb/block-figma-embed-block', {
 			setEditMode( state => ! state );
 		};
 
-		const toolbar = <PluginToolbar className={ `${ className }-toolbar` } isEditMode={ isEditMode } onEditModeToggle={ handleEditModeToggle } />;
-		const sidebar = <PluginSidebar className={ `${ className }-sidebar` } aspectRatio={ aspectRatio } onAspectRatioChange={ handleAspectRatioChange } />;
```

Comment:
moved these to be hardcoded in the component because when you add a class under the "Advanced" accordion, the generated class name is `wp-block-cgb-block-figma-embed-block foobar-sidebar` not `wp-block-cgb-block-figma-embed-block-sidebar` which breaks styling of the sidebar.